netbox/utilities/forms/widgets.py

```diff
@@ -116,6 +118,17 @@ class SelectDurationWidget(forms.NumberInput):
     template_name = 'widgets/select_duration.html'
 
 
+class MarkdownWidget(forms.Textarea):
+    template_name = 'widgets/markdown_input.html'
+
+
+class SmallMarkdownWidget(MarkdownWidget):
```

Comment:
I don't think this is actually needed. The `Smalltextarea` widget was a remnant of the old UI that I've coincidentally already ripped out for v3.5. We should be able to just stick with a single `MarkdownWidget` class.

Reply:
Should CommentFields that used SmallTextArea then just use the MarkdownWidget or should I change them back to using SmallTextArea?